doc/developer/debugging.rst

```diff
@@ -107,6 +164,25 @@ Add in the ``docker-compose.yaml`` file, in the ``geoportal`` service the follow
          - <c2cgeoportal_git_root>/geoportal/c2cgeoportal_geoportal:/opt/c2cgeoportal_geoportal/c2cgeoportal_geoportal
          - <c2cgeoportal_git_root>/geoportal/c2cgeoportal_admin:/opt/c2cgeoportal_geoportal/c2cgeoportal_admin
 
+Expose a service
+................
+
+To expose a service out of the docker-compose you can add a port for the service in the vars, e.g.:
+
+.. code:: yaml
+
+   vars:
+     docker_services:
+       <service>:
+         port: 8086
+
+Be careful one port can be open only one time on a server. To avoid that you can do a curl directly from a
```

Comment:
I don't understand how doing a curl helps to avoid the "one port can be open only one time on a server". What are we avoiding?

Reply:
When you do a curl in a container you can directly access to all the port of all the other container of the composition, than you didn't needs to publish them on the main host....

Reply:
=> Within the Docker composition you can access a port of a container, you can achieve this via curl.
This way, you do not need to publish this port on the main host.
:-)